### PR TITLE
added 3 true/false questions as quiz

### PR DESCRIPTION
**Pull request by elevran**

### CODE REVIEW

File: workshop/exercise-7/README.md [hunk 1]
```diff
@@ -90,9 +90,9 @@ Note that `cert-chain.pem` is Envoy’s public certificate (i.e., presented to t
 
 If, for some reason, mTLS is not functional, the Guestbook application will not operate correctly. This would be evident in its UI (e.g., it would display a [`Waiting for database connection`](waiting_on_database.png) message). In that case, mTLS should be disabled.
 
-mTLS it can be disabled globally or per service.
+mTLS can be disabled globally or per service.
 
-* To disable mTLS for all services in the mesh, comment out the `authPolicy` settings in the mesh configuration:
+* To disable mTLS for all services in the mesh, comment out the `authPolicy` settings in the mesh configuration, and then restart Istio Pilot to pick up the new configuration:
```
Comment (linsun) [inline workshop/exercise-7/README.md]: This instruction doesn't work.  It used to work in older istio release (like 0.2 or 0.3) but Dan Berg tried it today and didn't work on 0.7.1

File: workshop/exercise-7/README.md [hunk 2]
```diff
@@ -138,6 +140,16 @@ spec:
 > Note that the annotations can also be used to gradually enable mTLS on individual services, 
 >
 
+## Quiz
+
+True or False?
+
+1. Istio Auth/Citadel provides each microservice with a strong, cryptographic, identity in the form of a certificate. The certificates' life cycle is fully managed by Istio.
+
+1. Istio provides microservices with mutually authenticated connections, without requiring application code changes.
```
Comment (linsun) [inline workshop/exercise-7/README.md]: would be great if you could provide answer to this. :)

**Comment by vwilburn** [comment]
@elevran Will you be able to respond to Lin's thoughts above?

File: workshop/exercise-7/README.md (hunk 1 above)
Comment (elevran) [reply workshop/exercise-7/README.md]: Works with 0.6 which was the target before KubeCon, IIRC. Will fix for 0.7.1

File: workshop/exercise-7/README.md (hunk 2 above)
Comment (elevran) [reply workshop/exercise-7/README.md]: Sure, will add [true / false] indication next to each, but then I'm not sure how this is a quiz if the answer is inlined next to the question ;-)

File: workshop/exercise-7/README.md (hunk 1 above)
Comment (elevran) [reply workshop/exercise-7/README.md]: I will do this as part of PR to #39

**Comment by linsun** [comment]
think this is covered by other PR, closing it.  @elevran pls reopen if i'm wrong.

**Pull request closed without merging**